js/wallet.js

```diff
@@ -48,10 +165,125 @@ class JS extends Wallet {
     mockery.disable()
   }
 
+  get walletResult () {
+    return Immutable.fromJS({
+      bootStamp: 1512939627058,
+      properties: {
+        wallet: {
+          altcurrency: 'BAT',
+          keyinfo: {
+            seed: this.seed
+          }
+        }
+      }
+    })
+  }
+
+  get balanceResponse () {
+    return Immutable.fromJS({
+      probi: '0',
+      altcurrency: this.currency,
+      balance: this.balance,
+      parameters: {
+        adFree: {
+          currency: this.currency,
+          fee: {
+            BAT: this.defaultContribution
+          }
+        }
+      },
+      addresses: Immutable.fromJS(this.addresses)
+    })
+  }
+
+  get walletPassphrase () {
+    return 'wasp broken strong analyst until tray olympic arrow input bicycle gun settle prepare tissue road try sustain husband width brave section obey country area'
+  }
+
+  get seed () {
+    return Buffer.from([
+      32,
+      87,
+      30,
+      26,
+      223,
+      56,
+      224,
+      31,
+      213,
+      136,
+      248,
+      95,
+      136,
+      56,
+      250,
+      78,
+      179,
+      121,
+      255,
+      162,
+      195,
+      39,
+      143,
+      136,
+      18,
+      140,
+      49,
+      216,
+      221,
+      154,
+      78,
+      173
+    ])
+  }
+
+  getInfo (state) {
+    return state.getIn(['ledger', 'info'])
+  }
+
+  getStatus (state) {
+    return state.getIn(['status'])
+  }
+
+  getPreferences (state) {
+    return state.getIn(['about', 'preferences'])
+  }
+
   createWallet () {
     this.ledger.enable(defaultAppState)
     return this.ledger.getSynopsis()
   }
+
+  // Like create wallet, except the state is returned
+  _createWallet () {
```

Comment:
Can we rename this one into something like `createWalletState`, because when I saw `_` I though that it was private function